cmd/eib/main.go

```diff
@@ -1,7 +1,85 @@
 package main
 
-import "fmt"
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"os"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/suse-edge/edge-image-builder/pkg/config"
+)
+
+const (
+	argConfigFile = "config"
+	argVerbose    = "verbose"
+)
+
+func init() {
+	log.SetFormatter(&log.TextFormatter{
+		FullTimestamp:    true,
+		QuoteEmptyFields: true,
+	})
+	log.SetOutput(os.Stdout)
+}
+
+func parseFlags(programName string, args []string) (imageConfig *config.ImageConfig, output string, err error) {
+	flags := flag.NewFlagSet(programName, flag.ContinueOnError)
```

Comment:
Flag sets are used when a command is expected to have subcommands. Are there such plans?

Reply:
Ah, no, at least not currently. It wasn't an intentional decision, just me misreading the API.